docs/ECLStandardLibraryReference/SLR-Mods/GetLogicalFileAttribute.xml

```diff
@@ -31,7 +31,10 @@
           <entry><emphasis>attrname</emphasis></entry>
 
           <entry>A null-terminated string containing the name of the file
-          attribute to return.</entry>
+          attribute to return. Possible values are recordSize, recordCount,
+          size, clusterName, directory, numparts, owner, description, ECL,
+          partmask, numparts, name, modified, or protected. The value is
+          case-sensitive.</entry>
```

Comment:
And 'format', 'job', 'checkSum', 'kind', 'csvSeparate', 'csvTerminate' and 'workunit'